### DISCUSSION TITLE
Octavia CLI enable schema evolution

### DISCUSSION THREAD

**Discussion by lucienfregosibodyguard** [airbytehq/airbyte#28327] [General]
Hello could you look a this issue ?

https://github.com/airbytehq/airbyte/issues/28191

It would be awesome if it was possible to enable « propagate changes » in airbyte connection through octavia cli 

Many thankS

**Comment by marcosmarxm** [comment]
Hello @lucienfregosibodyguard whatever the team prioritize the issue we're going to get in touch. Right now this isn't in the roadmap but you can make a community contribution.

**Reply by lucienfregosibodyguard** [reply]
@marcosmarxm i understand that is not in the roadmap but that's a pity that the schema evolution feature is fully implemented on airbyte side but not in octavia side ..
